qa/vagrant/src/test/resources/packaging/tests/example/scripts/is_guide.painless

```diff
@@ -1 +1,5 @@
-_source['title'] == 'Elasticsearch - The Definitive Guide'
+def min = 300;
```

Comment:
`return doc.pages.value >= (params.min_num_pages ?: 300)` is the way I expect/hope folks will write this.